lib/Gemfile

```diff
@@ -40,6 +40,7 @@ gem "uuidtools",            "~>2.1.3",       :require => false
 gem "sass",                 "3.1.20",        :require => false
 gem "trollop",              "~>1.16.2",      :require => false
 gem "psych",                "~>2.0.10"
+gem "foreman_api",          "~>0.1.11",      :require => false  # used by lib/manageiq_foreman
```

Comment:
We can't reference it as a gem (build reasons), so somehow it has to be referenced here.